app/containers/ExecutionNewPage.js

```diff
@@ -55,13 +55,15 @@ import {
   DatasetSiteIcon,
   DatasetSessionIcon,
   SchedulerIcon,
+  SchedulerParamIcon,
   NextIcon,
   PreviousIcon,
 } from '../components/icons'
 
 import Modal from '../components/Modal'
 import Content from '../components/Content'
 import Box from '../components/Box'
+import { default as SummaryCard} from '../components/ExecutionNewPageSummary'
```

Comment:
`import SummaryCard from '../components/ExecutionNewPageSummary'` works the same way!

the `default as` is mainly for exporting something with a different name.